CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 0.10.8dev
 * [Fix] Fix edge case where `select` and other SQL keywords were not properly used to find where the user's query started, causing argument parsing issues (#973)
+* [Fix] Ensure config is always read from correct section in `pyproject.toml` and `~/.jupysql/config/` ([#969](https://github.com/ploomber/jupysql/issues/969), [#975](https://github.com/ploomber/jupysql/issues/975)))
```

Comment:
I've added the `no-changelog` label to this PR. Please remove this entry now because the change is an internal one.